fix: updated roks quickstart ref arch

Khuzaima05 opened the pull request:
### Description

Changes made: [ref](https://github.ibm.com/ibmcloud/ibm-cloud-stencils/tree/master?tab=readme-ov-file#20-examples)

- Renamed "VSI" to "subnet" for clarity
- Updated service names: "Key Management" → "Key Management (Key Protect / Existing Hyper Protect Crypto Service)
- "Object Storage" as "Cloud Object Storage" -> "Object Storage"
- resource group in dotted box
- Coloured box for subnet
- some colour changes


issue: https://github.ibm.com/GoldenEye/issues/issues/15100

### Release required?
<!--- Identify the type of release. For information about the changes in a semantic versioning release, see [Release versioning](https://terraform-ibm-modules.github.io/documentation/#/versioning). --->

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

<!--- If a release is required, replace this text with information that users need to know about the release. Write the release notes to help users understand the changes, and include information about how to update from the previous version.

Your notes help the merger write the commit message for the PR that is published in the release notes for the module. --->

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

Khuzaima05 commented:
These are the changes made: [ref](https://github.ibm.com/ibmcloud/ibm-cloud-stencils/tree/master?tab=readme-ov-file#20-examples)
- Renamed "VSI" to "subnet" for clarity
- Updated service names: "Key Management" → "Key Protect
- "Object Storage" as "Cloud Object Storage"
- resource group in dotted box 
- Coloured box for subnet

Khuzaima05 commented:
/run pipeline

Khuzaima05 commented:
/run pipeline

Khuzaima05 commented:
/run pipeline

vburckhardt commented:
You're getting only one cluster however in quick start. With 2 nodes. New diagram is showing 2 clusters.

vburckhardt commented:
Also, one of the sticking points is that there is no compute resource in the management VPC. We should explicitly indicate that the empty subnet is intentionally left vacant, allowing the user to deploy their preferred compute—whether VSI or OpenShift—for management purposes, in alignment with the FsCloud topology. Possibly put "Reserved for user-managed compute (per FsCloud topology)"

On the HPCS - do quick start support it? If not, remove HPCS from diagram.

Could we also loop back with Jose Monge to validate the new version. He's driving the stencil initiative and has been historically involved with our group.

Khuzaima05 commented:
@vburckhardt 

- Management VPC: Added a comment inside the subnet stating "Reserved for user-managed compute (per FsCloud topology)" to clarify that it's intentionally left vacant for user-managed VSI or OpenShift deployments.

- HPCS: In the quick start configuration, [hpcs is hardcoded as false](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/46a8fdc0798023ad2efe315473788b013541c6c0/patterns/roks-quickstart/main.tf#L94C6-L94C30), meaning only Key Protect is supported for encryption. I've updated the diagram accordingly to reflect Key Protect only.

- Cluster in Workload VPC: The configuration defines a single cluster with two worker nodes. The earlier diagram showed two separate clusters, which was misleading. I’ve updated it to show one cluster spanning both zones, with a worker node icon in each subnet.

Reviewed the updated diagram with Jose Monge . He confirmed the diagram looks good and suggested a few minor changes, which I’ve incorporated.

Khuzaima05 commented:
/run pipeline

vburckhardt commented:
I believe it’s acceptable to proceed. Additionally, we should update the diagrams in the other variations (particularly the standard version) to align with the new version of the quick start. Most users prefer to quickly compare variations using the diagrams - could we make sure that this is tracked and prioritized.
